src/app/query/hiro-config/hiro-config.query.ts

```diff
@@ -35,9 +35,10 @@ interface HiroConfig {
   feeEstimationsMinMax?: FeeEstimationsConfig;
 }
 
-// TODO: Change back to 'main' before merging
-const GITHUB_PRIMARY_BRANCH = 'dev';
-const githubWalletConfigRawUrl = `https://raw.githubusercontent.com/${GITHUB_ORG}/${GITHUB_REPO}/${GITHUB_PRIMARY_BRANCH}/config/wallet-config.json`;
+const GITHUB_PRIMARY_BRANCH = 'main';
+const githubWalletConfigRawUrl = `https://raw.githubusercontent.com/${GITHUB_ORG}/${GITHUB_REPO}/${
+  BRANCH ?? GITHUB_PRIMARY_BRANCH
```

Comment:
@kyranjamie is this the best way to get the current branch for the config?

Reply:
I think this should be fine, yeah

Reply:
I'd rename the default now, to mean default or fallback

Reply:
BRANCH doesn't seem to be working here, hence the failed test bc now Coinbase should be listed first based on the wallet-config but MoonPay appears to still be listed first (which is on main).

Reply:
I need to check this again today, but it was `undefined`.

Reply:
It will be locally, see webpack config, but should work for other envs

Reply:
Yep, but I did expect the test to pass here. I will push up a log in the code and test the PR build today.

Reply:
It is this, obvious now bc in header `refs/pull/2560/merge`. So, I'll look at modifying it for what we need here.

Reply:
Figured it out, we have to use the Github env variable available `process.env.GITHUB_HEAD_REF`. For `BRANCH` we are using, `process.env.GITHUB_REF`. I made a new global variable `BRANCH_NAME`.